check_link.rb

```diff
@@ -5,7 +5,10 @@
 require 'optparse'
 
 class LinkChecker
-  INTERNAL_LINK_REGEXP = /\[[\.\w\s\-]+\]\(([\w\/\-\#]*)\)/
+  INTERNAL_LINK_REGEXPS = [
+    /\[[\.\w\s\-]+\]\(([\w\/\-\#]*)\)/,
+    /<a href="([\/core|\/sdk][\w\/\-\#]*)">/
```

Comment:
```suggestion
    /<a href="\/[\w\/\-\#]*)">/
```

Reply:
You forgot the `(` to capture the link but apart from that :+1: